src/console_cmds.cpp

```diff
@@ -1894,6 +1894,18 @@ static void IConsoleDebugLibRegister()
 	IConsoleAliasRegister("dbg_echo2",      "echo %!");
 }
 #endif
+DEF_CONSOLE_CMD(ConFramerate)
+{
+	extern void ConPrintFramerate(); // framerate_gui.cpp
+
+	if (argc == 0) {
```

Comment:
Considering adding a way to summon the framerate window from console, so you can also view it on the title screen.